AI-and-Analytics/End-to-end-Workloads/Census/census_modin.ipynb

```diff
@@ -306,9 +306,9 @@
    "name": "python",
```

Comment:
same here. possible to remove this changes for census if it is only for version? 
you could checkout the file before you create a commit.